crowdin/messages.json

```diff
@@ -1549,7 +1549,6 @@
   "1730081262": "Spot your favourite brand?",
   "1733606989": "0.50",
   "1733660945": "Funds available as soon as confirmed",
-  "1734442908": "CFDs and other products offered on this website are complex instruments with high risk of losing money rapidly owing to leverage. <0>{{loss_percent}}% of retail investor accounts lose money when trading CFDs with Deriv</0>. You should consider whether you understand how these products work and whether you can afford to risk losing your money.",
```

Comment:
Is this also part of the change 👀

Reply:
No @adrienne-deriv , but every PR has it. It's from `upstream/master`

Reply:
okiess